packages/sit-onyx/src/components/OnyxDataGrid/features/HeaderCell.vue

```diff
@@ -26,7 +26,8 @@ const slots = defineSlots<{
   align-items: center;
   gap: var(--onyx-spacing-2xs);
 
-  &__actions {
+  &__actions,
+  &__remove-actions {
```

Comment:
```suggestion
  &__actions {
```

There is no remove-actions class, or?